bertytech/content/newsletter/news-60/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Berty News #60"
+date: 2021-06-11
+aliases: ["/newsletter/weekly-60"]
+---
+
+# News #60
+
+Hi cool kids 🏴☠️
+
+Here's a newsletter that will make you happy: Berty is on TestFlight and will be soon on the Play Store.🤩🤩🤩🤩. It's coming soon. We know we announced in our previous newsletter that It was supposed to arrive last week. But there is a little delay in the validation and some bugs to fix. Anyway, if you have an iPhone, you can download the alpha! (Read below how!)
+
+Just a quick word about the team before letting you dive into this newsletter: everyone is doing well. We admit that the restaurants and bars are finally open again until 11pm. It feels so good to be back to normal. We can't wait to celebrate the release of our Alpha community. We are very excited to showcase our work. 😊
```

Comment:
`We admit that the restaurants and bars are finally open again until 11pm.`

What the meaning of that? A word is missing maybe?

Reply:
`Play Store.🤩🤩🤩🤩.` -> `Play Store 🤩🤩🤩🤩.`

Reply:
`that It was supposed` -> `that it was supposed`

Reply:
`download the alpha` -> `download the Alpha` (consistency)